### PR TITLE
Made changes in paramGeneration for code and OcTsx

### PR DESCRIPTION
**Pull request by rishabh-ti**

### CODE REVIEW

File: src/utils/code.ts [hunk 1]
```diff
@@ -48,13 +48,17 @@ const buildParams = (paramsName: any) => {
...
     const optional = param.optional ? '?' : ''
     paramTypes += `${param.name}${optional}: ${param.type}, `
-    let operand =
-      param.type == 'string'
-        ? `'${param.value}'`
-        : param.type == 'Function'
-        ? `${param.value.slice(1, param.value.length - 1)}`
-        : `${param.value}`
-    params += `${param.name}=${operand}, `
+    if (ocTsx) {
```
Comment (vinayak-sachdeva) [inline src/utils/code.ts]: you are adding default value when ocTsx is true. Which means the file inside bit component right?
But we wanted the opposite

Comment (rishabh-ti) [reply src/utils/code.ts]: No, there are two functions - generateCode which generates the OcTsx in bit component and generateOcTsx which generates OcTsx in custom-components.

Comment (rishabh-ti) [reply src/utils/code.ts]: Improved naming and fixed minor error in latest commit

File: src/utils/code.ts [hunk 2]
```diff
@@ -48,13 +48,18 @@ const buildParams = (paramsName: any) => {
...
     paramTypes += `${param.name}${optional}: ${param.type}, `
-    let operand =
-      param.type == 'string'
-        ? `'${param.value}'`
-        : param.type == 'Function'
-        ? `${param.value.slice(1, param.value.length - 1)}`
-        : `${param.value}`
-    params += `${param.name}=${operand}, `
+    params += `${param.name}`
+    if (customOcTsx || !param.optional) {
```
Comment (vinayak-sachdeva) [inline src/utils/code.ts]: whats the benefit of giving default value when param is not optional?

**Review by vinayak-sachdeva — APPROVED** [review]
Also rename the defaultValue to previewValue in panel

**Pull request merged**